mrbgems/mruby-error/test/exception.c

```diff
@@ -1,6 +1,7 @@
 #include <mruby.h>
 #include <mruby/error.h>
 #include <mruby/array.h>
+#include <mruby/presym.h>
```

Comment:
The only difference between the two versions of `MRB_E_SYM()` is slight performance. So unless the source file uses `MRB_SYM()` in it (as described in `doc/guides/symbol.md`), I don't think we need to include `mruby/presym.h` (to avoid unnecessary recompilation).